test/e2e/resize_nodes.go

```diff
@@ -330,7 +330,14 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	// and cause it to fail if DNS is absent or broken.
 	// Use the IP address instead.
 
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump DROP", testContext.CloudConfig.MasterName)
+	destination := testContext.CloudConfig.MasterName
+	if providerIs("aws") {
+		// TODO: Use for all clouds?
+		// TODO: Avoid hard-coding this
+		destination = "127.20.0.9"
```

Comment:
It is the Master internal IP on AWS: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/aws/util.sh#L51-L53

I guess this could use a comment.  Or I could try changing the name passed in to the e2e tests to one that is resolvable.  Or perhaps you have an even better idea?